### PR TITLE
WIP: fix bundling

### PR DESCRIPTION
**Pull request by carlbrugger**

### CODE REVIEW

File: plugins/autocast/package.json [hunk 1]
```diff
@@ -53,15 +53,15 @@
...
   "peerDependencies": {
-    "@flatfile/api": "^1.7.4",
-    "@flatfile/listener": "^1.0.1"
+    "@flatfile/api": "^1.8.0",
+    "@flatfile/listener": "^1.0.1",
+    "@flatfile/plugin-record-hook": "^1.5.1"
   },
   "devDependencies": {
-    "@flatfile/api": "^1.7.4",
+    "@flatfile/plugin-record-hook": "^1.5.1",
```
Comment (bangarang) [inline plugins/autocast/package.json:64]: Shouldn't this only be in `peerDependencies`?
```suggestion
```

File: plugins/constraints/package.json [hunk 2]
```diff
@@ -47,14 +47,14 @@
...
   "peerDependencies": {
-    "@flatfile/api": "^1.7.4",
-    "@flatfile/listener": "^1.0.1"
+    "@flatfile/api": "^1.8.0",
+    "@flatfile/listener": "^1.0.1",
+    "@flatfile/plugin-record-hook": "^1.5.1"
   },
   "devDependencies": {
-    "@flatfile/api": "^1.7.4",
+    "@flatfile/plugin-record-hook": "^1.5.1",
```
Comment (bangarang) [inline plugins/constraints/package.json:57]: Shouldn't this only be in `peerDependencies`?
```suggestion
```

**Review by bangarang — APPROVED** [review]

File: plugins/autocast/package.json (hunk 1 above)
Comment (carlbrugger) [reply plugins/autocast/package.json:64]: No, turbo build won't realize that a plugin is dependent on another plugin unless the dependency is either in `deps` or `devDeps`.

File: plugins/constraints/package.json (hunk 2 above)
Comment (carlbrugger) [reply plugins/constraints/package.json:57]: Same as above.

**Pull request merged**